File: src/medrelay/enums/resource_type.py
from enum import StrEnum


class ResourceType(StrEnum):
    AMBULANCE = "AMBULANCE"
    PATROL = "PATROL"
    TEAM_LEAD = "TEAM_LEAD"
    BUGGY = "BUGGY"
    OTHER = "OTHER"

    def label(self) -> str:
        return {
            ResourceType.AMBULANCE: "Ambulance",
            ResourceType.PATROL: "Patrol",
            ResourceType.TEAM_LEAD: "Team Lead",
            ResourceType.BUGGY: "Buggy",
            ResourceType.OTHER: "Other",
        }[self]

File: app/Enums/ResourceType.php
<?php

declare(strict_types=1);

namespace App\Enums;

enum ResourceType: string
{
    case Ambulance = 'ambulance';
    case Patrol = 'patrol';
    case TeamLead = 'team_lead';
    case Buggy = 'buggy';
    case Other = 'other';

    public function label(): string
    {
        return match ($this) {
            self::Ambulance => 'Ambulance',
            self::Patrol => 'Patrol',
            self::TeamLead => 'Team Lead',
            self::Buggy => 'Buggy',
            self::Other => 'Other',
        };
    }
}
